tests/language/loop.liq

```diff
@@ -0,0 +1,42 @@
+#!../../src/liquidsoap ../../libs/pervasives.liq
+
+%include "test.liq"
+  
+success = ref(true)
+
+def t(x, y)
+  if x != y then
+    print("Failure: got #{x} instead of #{y}")
+    success := false
+  end
+end
+  
+def f() =
+  n = ref(1)
+  while !n < 10 do
+    n := !n * 2
+  end
+  t(!n, 16)
+
+  n = ref(0)
+  for i in 11 do
```

Comment:
Do we really want to make `int` iterable? This looks weird to me, I think it'll be confusing.

Reply:
I have removed the `Iterable` type constraint for now, it was buggy... Support for iterating over integers has gone too for now!